site/guide/model-validation/add-update-model-findings.qmd

```diff
@@ -0,0 +1,142 @@
+---
+title: "Add and update model findings"
+date: last-modified
+---
+
+Add findings within the {{< var validmind.platform >}} at the model or documentation section level, and update your findings to include proposed remediation plans and supporting documentation.
+
+::: {.prereq}
+
+## Prerequisites
+
+- [x] {{< var link.login >}}
+- [x] The model is already registered in the model inventory.[^1]
+- [x] Model documentation is completed or in progress for your model.[^2]
+- [x] You are a `Validator` or assigned another role with sufficient permissions to perform the tasks in this guide.[^3] 
+
+:::
+
+## Add model findings
+
+{{< include _add-model-findings.qmd >}}
+
+#### Add finding via overview
+
+To log a finding from the documentation overview: 
+
+1. On your model's **Documentation** page, click **{{< fa plus >}} Add Finding**.
+
+2. On the **Add Model Finding** page that opens, provide information for: 
+
+   - Title
+   - Risk area
+   - Owner
+   - Severity
+   - Due date
+   - Documentation section
+   - Description
+
+   <!-- At a minimum, you must provide the required information indicated by `*`. -->
```

Comment:
Very minor, I'd delete this and the other occurrence of commented out text. I don't think we should explain that `*` means something is required.

Reply:
This is actually no longer true as well, since the * was removed from the UI. I've mentioned it and they say it shouldn't have been removed but I agree it doesn't need to be pointed out (should be obvious!).